remoteappmanager/static/js/vue/filters.js

```diff
@@ -1,20 +1,20 @@
 define([
     "components/vue/dist/vue",
     "urlutils"
-], function (Vue, urlutils) {
+], function (Vue, urlUtils) {
```

Comment:
question is if we should use the same for modules, or just stick to the filename (all lowercase).

But we use Vue... but then again, Vue is also weird because it's a class...